recipe/meta.yaml

```diff
@@ -1,58 +1,73 @@
 {% set name = "prefect" %}
-{% set version = "1.2.4" %}
+{% set version = "2.0.1" %}
 
 package:
   name: {{ name|lower }}
   version: {{ version }}
 
 source:
   url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
-  sha256: 774a50684cd8ef365d071ed1a4f2249213559fed40e7a0fd9ef36a51009e3064
+  sha256: b068d6446ab7c227f703e4670862274ce3eea14e2925a41eae48c184fdf32ce2
 
 build:
-  noarch: python
+  skip: True  # [py==38 and py==310]
```

Comment:
Can you explain this one?

Reply:
It was used to skip builds with python versions of 3.8 and 3.10 in order to not download the affiliated packages per the Orion [requirements.txt](https://github.com/PrefectHQ/orion/blob/main/requirements.txt) file